src/test/java/de/dhbw/karlsruhe/bottom/up/BottomUpAcceptorTest.java

```diff
@@ -0,0 +1,130 @@
+package de.dhbw.karlsruhe.bottom.up;
+
+import com.google.gson.Gson;
+import de.dhbw.karlsruhe.bottom.up.models.BottomUpAcceptor;
+import de.dhbw.karlsruhe.bottom.up.validation.BottomUpAcceptorValidation;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.io.*;
+import java.util.Scanner;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+ class BottomUpAcceptorTest {
+
+    @Test
+     void checkCorrectBottomUpAcceptorValidationTest() throws FileNotFoundException {
+        String grammarAsJson = getGrammarAsJson(
+                "src/test/resources/derivation_tree/grammarCorrect1.json");
+
+        Gson gson = new Gson();
+
+        try (Reader reader = new FileReader(
+                "src/test/resources/bottom_up/bUAcceptorCorrect1.json")) {
+            BottomUpAcceptor buAcceptor = gson.fromJson(reader, BottomUpAcceptor.class);
+            BottomUpAcceptorValidation bUAcceptorValidation = new BottomUpAcceptorValidation(grammarAsJson);
+            String word = "(())";
+            assertTrue(bUAcceptorValidation.checkAcceptor(buAcceptor, word));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+
+    @Test
+     void checkNullWordBottomUpAcceptorValidationTest() throws IOException {
+        String grammarAsJson = getGrammarAsJson(
+                "src/test/resources/derivation_tree/grammarCorrect1.json");
+
+        Gson gson = new Gson();
+
+        try (Reader reader = new FileReader(
```

Comment:
Try-Catch muss noch abgeändert werden. Wenn ein Fehler fliegen sollte, wird der Test trotzdem als korrekt erkannt. 
-> Muss in allen Tests angepasst werden

Reply:
Habe es so angepasst, dass es wie im DerivationTreeValidationTest funktioniert.

Reply:
Beim DerivationTreeValidationTest sollte es momentan leider auch nicht funktionieren. Mein erster Ansatz wäre eine Schaltung in assertNotThrows(...) zu machen. Würde aber noch mal nachschauen, ob es vielleicht eine bessere Alternative gibt.

Reply:
Ich habe den DerivationTreeValidationTest ausprobiert und da scheitert der Test, wenn es die Datei nicht findet (der Pfad falsch ist). Daher habe ich es genau so implementiert :)

Reply:
Alles klar!